### PR TITLE
Make new index page

### LINKED ISSUE #43: Make the landing page good to look.
Currently we just have a vague implementation of our landing page.
screen shot 2018-01-30 at 6 20 17 pm

Please propose a good implementation.

### PR DESCRIPTION
**Pull request by littlestar642**
fix #43

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,7 @@
   "description": "Clone of omegle.com",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "postinstall": "./node_modules/bower/bin/bower install"
```
Comment (sumedh123) [inline package.json]: why did you remove this?

File: public/css/styles.css [hunk 2]
```diff
@@ -33,15 +33,34 @@
...
+  text-shadow: 0 0 10px whitesmoke;
+}
+.add{
+  margin: 40px;
+  padding: 10px;
+}
+.topClear{
+  top:25vh;
+}
+.body{
```
Comment (sumedh123) [inline public/css/styles.css]: too unclean code.

Comment (littlestar642) [reply public/css/styles.css]: What does this mean?

File: package.json [hunk 3]
```diff
@@ -20,7 +20,6 @@
   "dependencies": {
     "bower": "^1.8.2",
     "express": "^4.16.2",
-    "faker": "^4.1.0",
```
Comment (sumedh123) [inline package.json]: why?

Comment (sumedh123) [reply package.json]: Please dont remove the things. Just make changes what you want.

**Comment by littlestar642** [comment]
https://nameless-stream-60765.herokuapp.com/
Deployment link

File: views/index.hbs [hunk 4]
```diff
@@ -1,38 +1,59 @@
...
+                <p class="sty add">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Error cupiditate optio illum laborum voluptatum reiciendis consequatur laboriosam maxime aspernatur deserunt ullam explicabo at veniam vitae numquam autem quis dolores, ad, nihil amet eaque voluptatibus? Ea officia assumenda dolorum, vel iste nostrum a unde beatae qui magni itaque maxime, accusamus tempore.</p>
+            </div>
+            <div class="column center aligned">
+                <a href="/chat">
+                <div class="ui button animated fade massive green inverted topClear">
```
Comment (sumedh123) [inline views/index.hbs]: indentation

File: public/css/styles.css [hunk 5]
```diff
@@ -59,4 +68,14 @@
     padding-left: 0px!important;
     padding-right: 0px!important;
   }
-}
+  .topClear{
```
Comment (sumedh123) [inline public/css/styles.css]: two classes with same name?

File: public/css/styles.css [hunk 6]
```diff
@@ -59,4 +68,14 @@
...
+    top:0vh;
+  }
+  .add{
```
Comment (sumedh123) [inline public/css/styles.css]: here too

File: views/index.hbs [hunk 7]
```diff
@@ -1,38 +1,59 @@
...
+                <h1 class="sty">WE HOPE YOU HAVE A GOOD TIME HERE</h1><br>
+            </div>
+            <div class="sixteen wide column">
+                <h3 class="sty">MADE WITH ❣ AND ☕</h3>
+                <h2 class="sty">OMECLONE</h2>
+            </div>
+        </div>
+    </div>
+</body>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (sumedh123) [inline views/index.hbs]: use bower components instead of cdn

File: public/css/styles.css (hunk 5 above)
Comment (littlestar642) [reply public/css/styles.css]: i made a media breakpoint giving the two classes the same name with different property value.

File: public/css/styles.css [hunk 8]
```diff
@@ -33,15 +33,24 @@
...
 }
-
 .hide{
   display: none!important;
 }
+.sty{
+  text-align: center;
+  text-shadow: 0 0 10px whitesmoke;
+}
+.add{
```
Comment (sumedh123) [inline public/css/styles.css]: follow style guide. add space between class name and {
